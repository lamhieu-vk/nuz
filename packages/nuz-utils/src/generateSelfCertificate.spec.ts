import * as func from './generateSelfCertificate'
import generateSelfCertificate from './generateSelfCertificate'

describe('generateSelfCertificate', () => {
  it('Exported as default', () => {
    expect(func.default).toBeDefined()
  })

  it('Should match to the snapshot', () => {
    const attributes = { name: 'awesomeNuz', value: 'nuz.app' }

    const result = generateSelfCertificate(attributes)
    expect(generateSelfCertificate(attributes)).toMatchSnapshot({
      cert: expect.any(String),
      fingerprint: expect.any(String),
      private: expect.any(String),
      public: expect.any(String),
    })
  })
})
