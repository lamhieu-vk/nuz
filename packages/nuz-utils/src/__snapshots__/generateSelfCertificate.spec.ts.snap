// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`generateSelfCertificate Should match to the snapshot 1`] = `
Object {
  "cert": Any<String>,
  "fingerprint": Any<String>,
  "private": Any<String>,
  "public": Any<String>,
}
`;
